l10n/en/banners/fundraising.ftl

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+banner-fundraising-title = The future is being decided online. Help { -brand-name-mozilla } fight for a web for all of us.
+banner-fundraising-body = Our personal data is collected every time we access the web, and it's not easy to find out what happens with it. Misinformation is making it harder for us to find the truth. Web-connected devices are pushed to market without meeting minimum security standards. The non-profit { -brand-name-mozilla-foundation } fights to make the internet safe, open and accessible to all. <strong>Will you donate today?</strong>
+banner-fundraising-monthly = Monthly
+banner-fundraising-one-time = One-time
+banner-fundraising-donate = Donate
+
+# Variables:
+#   $amount (number) - Inserts a number in US dollars, e.g. '$50'. Adapt the string in your translation for your locale conventions if needed, ex: { $amount } US$
+# You can also localize the currency (e.g. to use € instead of $), but only if your currency is available on https://donate.mozilla.org (check the currency dropdown)
```

Comment:
Nit: Suggest keeping this comment all on one line since it's related to the variable?

Reply:
Actually, technically you're correct it's not related to the variable but to the surrounding string instead. So maybe this does make the most sense already 👍